app/_kong_plugins/mocking/examples/basic-configuration.yaml

```diff
@@ -0,0 +1,125 @@
+title: Pass an OpenAPI spec to the plugin through decK
+description: |
```

Comment:
The goal of this is actually to mock endpoints using an OpenAPI spec.
Can you update the file name, title and description accordingly? (something along those lines)

deck doesn't need to be part of the title, the platform generates